content/en/docs/combined-matrix.md

```diff
@@ -0,0 +1,268 @@
+| Amazon AWS    | EC2, ECS, EKS                                                   |                       | All supported versions |       |
+| Cloud Foundry | PKS <ul><li>Versions A.B - X.Y</li></ul>                        |                       | All supported versions |       |
+| Google Cloud  | App Engine, Compute Engine, GKE                                 |                       | All supported versions |       |
+| Kubernetes    | Manifest-based deployments <ul><li>Versions A.B - X.Y</li></ul> |                       | All supported versions |       |
+## Dynamic accounts
+
+![Generally available](/images/ga.svg) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+
+Dynamic accounts (external account configurations) for Spinnaker allow you to manage account configuration outside of Spinnaker, including secrets. Armory Enterprise supports the following backend providers:
+
+#### Backend provider
+
+| Provider | Version | Armory Enterprise | Notes |
+|---------|---------|---------|---------|
+| Git  | x.y.z | All supported versions |
+| S3  | n/a | All supported versions |
+| Vault  | x.y.z | All supported versions |
+
+#### Supported Spinnaker services
+
+| Service     | Account types               | Note                                                                                                           |
+|-------------|-----------------------------|----------------------------------------------------------------------------------------------------------------|
+| Clouddriver | Cloud provider, artifact    | Automatic configuration refreshing is supported for Cloud Foundry and Kubernetes cloud provider accounts only. |
+| Echo        | Pub/Sub                     |                                                                                                                |
+| Igor        | CI systems, version control |                                                                                                                |
+
+## External storage
+
+![Generally available](/images/ga.svg) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+
+Spinnaker requires an external storage provider for persisting application settings and pipelines. Armory supports the following storage solutions:
+
+| Provider              | Armory Enterprise      | Notes |
+|-----------------------|------------------------|-------|
+| Azure Storage         | All supported versions |       |
+| Google Cloud Storage  | All supported versions |       |
+| Minio                 | All supported versions |       |
+| S3                    | All supported versions |       |
+| Oracle Object Storage | All supported versions |       |
+
+## Observability
+
+![Generally available](/images/ga.svg) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+
+{{% alert %}}Armory Enterprise 2.20 and later require the Observability plugin for monitoring Spinnaker. Versions earlier than 2.20 use the Spinnaker monitoring daemon, which is deprecated. {{% /alert %}}
```

Comment:
@brian-armory does the Observability plugin already support Datadog, ELK, and Splunk? I thought that work wasn't done yet? If not we need to be careful about what we say about support for these providers in 2.20.x

Reply:
The Managed team owns the monitoring daemon, so I'm guessing they own the Observability Plugin too. Can you comment @jasonmcintosh?